styles/faas.css

```diff
@@ -261,7 +321,13 @@
   margin-right: calc(var(--grid-margins-width) * -1 / 2);
 }
 
+/* tablet up */
```

Comment:
Adding comments for breakpoints seems superfluous

Reply:
I'll remove the comments